pkg/operator/staticpod/controller/missingstaticpod/missing_static_pod_controller.go

```diff
@@ -0,0 +1,260 @@
+package missingstaticpodcontroller
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/informers"
+	corelisterv1 "k8s.io/client-go/listers/core/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	"github.com/openshift/library-go/pkg/operator/staticpod/internal"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+type missingStaticPodController struct {
+	operatorClient                    v1helpers.StaticPodOperatorClient
+	podListerForTargetNamespace       corelisterv1.PodNamespaceLister
+	configMapListerForTargetNamespace corelisterv1.ConfigMapNamespaceLister
+	targetNamespace                   string
+	operandName                       string
+
+	lastEventEmissionPerNode lastEventEmissionPerNode
+}
+
+type lastEventEmissionPerNode map[string]struct {
+	revision  int
+	timestamp time.Time
+}
+
+// New checks the latest static pods pods and uses the installer to get the
+// latest revision.  If the installer pod was successful, and if it has been
+// longer than the terminationGracePeriodSeconds+120seconds since the installer
```

Comment:
nit: `terminationGracePeriodSeconds+120 seconds

Reply:
that is already what is written no?

Reply:
yes, `120seconds`-> `120 seconds` :)